### DISCUSSION TITLE
Trigger base alarm

### DISCUSSION THREAD

**Discussion by ianchapplethor** [samemory/homebridge-eufy-security-control#296] [Q&A] [answered]
Hi is it possible to trigger the alarm on the base station ?

My scenario is that i have 2 Base's due to camera range and i have all the alarm stuff (door contacts/keypad and so on) all hooked up to one base. What i would like to do is when when the alarm is triggered on base, set off the alarm on Base 2.

Is that possible or does anyone have another suggestion of how to achieve that ?

Thanks

Ian

**Comment by mpjalexander** [comment]
I’d also like to see this feature added.

**Comment by schliemann** [comment]
Should be possible.

**Comment by lenoxys** [comment, marked as answer]
Like the idea please register a feature request here : https://github.com/samemory/homebridge-eufy-security/issues/new/choose
